docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Aiven for ClickHouse® data service integration
+sidebar_label: Data integration
+keywords: [data service integration, data source integration, managed credentials integration, managed databases integration, named collections]
+---
+
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons";
+
+Aiven for ClickHouse® supports different types of integration allowing you to efficiently connect with other services or data sources and access the data to be processed.
+
+There are a few ways of classifying integration types supported in Aiven for ClickHouse:
+
+- [By purpose](/docs/products/clickhouse/concepts/data-integration-overview#observability-integrations-vs-data-source-integrations):
+  observability integration vs data source integration
+- [By location](/docs/products/clickhouse/concepts/data-integration-overview#in-aiven-integrations-vs-external-integrations):
+  in-aiven integration vs external integration
+- By scope: [managed databases integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration) and
+  [managed credentials integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Observability integrations vs data source integrations
+
+Aiven for ClickHouse supports observability integrations and data source integrations,
+which have different purposes:
+
+- [Observability integration](/docs/products/clickhouse/howto/list-integrations) is
+  connecting to other services (either in-Aiven or external) to expose and process logs
+  and metrics.
+- Data service integration is connecting to other services (either in-Aiven or external)
+  to use them as data sources. In Aiven for ClickHouse, data service integration is
+  possible with the following data source types:
+
+  - Apache Kafka®
+  - PostgreSQL®
+  - MySQL®
+  - ClickHouse®
+  - Amazon S3®
+
+## In-Aiven integrations vs external integrations
```

Comment:
`in-Aiven` as an adjective sounds off to me. Does this appear in the UI? Why not `Internal` ?